test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -29,6 +32,29 @@
 import seedu.addressbook.data.tag.UniqueTagList;
 
 public class TestUtil {
+    @Test
+    public void isAnyNull() throws Exception {
+        // No false positives
```

Comment:
Maybe describe the purpose of the test cases (e.g. empty string should not be null, second parameter null should still return have null).